web/viewer.css

```diff
@@ -352,7 +352,7 @@ html[dir='rtl'] #outerContainer.sidebarOpen #viewerContainer:not(.pdfPresentatio
   background-color: var(--sidebar-bg-color);
 }
 html[dir='ltr'] #toolbarSidebar {
-  box-shadow: inset -1px 0 0 rgba(0, 0, 0, 0.25), 
```

Comment:
I didn't manually change this line, but upon saving my editor removed this trailing space; given that we shouldn't have trailing spaces anywhere keeping this change thus seem OK here.